frontend/src/metabase/query_builder/components/notebook/steps/FilterStep/FilterStep.tsx

```diff
@@ -84,6 +84,7 @@ export function FilterStep({
             onClose={onClose}
           />
         )}
+        withLegacyPopover
```

Comment:
without this (which I copied from AggregationStep) the popover was aggressively closing whenever you clicked an expression suggestion

Reply:
ok, removed that commit